openpype/tools/publisher/window.py

```diff
@@ -388,6 +388,20 @@ def __init__(self, parent=None, controller=None, reset_on_show=None):
     def controller(self):
         return self._controller
 
+    @property
+    def reset_on_show(self):
+        return self._reset_on_show
+
+    @reset_on_show.setter
+    def reset_on_show(self, value):
+        self._reset_on_show = value
+
+    def set_comment_input_text(self, text=""):
+        self._comment_input.setText(text)
```

Comment:
I'm pretty sure this should go through the controller - not just the UI element. @iLLiCiTiT what do you think?

Reply:
I think they are not alternatives 
![image](https://github.com/ynput/OpenPype/assets/20871534/5e98e0d3-ef92-4a8f-a434-0a1a56a53271)

Also, most code in publisher depends on `self._comment_input` when updating the comment field in the UI